packages/fxa-settings/src/pages/InlineTotpSetup/index.tsx

```diff
@@ -0,0 +1,397 @@
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import classNames from 'classnames';
+import { MozServices } from '../../lib/types';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { useFtlMsgResolver } from '../../models';
+import { logViewEvent, logPageViewEvent } from '../../lib/metrics';
+import { ReactComponent as TwoFactorImg } from '../Signin/SigninTotpCode/graphic_two_factor_auth.svg';
+import InputText from '../../components/InputText';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+
+type InlineTotpSetupProps = {
+  code: string;
+  serviceName?: MozServices;
+};
+
+export const InlineTotpSetup = ({
+  code,
+  serviceName,
+}: InlineTotpSetupProps) => {
+  logPageViewEvent('inline-totp-setup', { entrypoint_variation: 'react' });
+
+  /*
+   * TODO:
+   *  - Write functionality to get TOTP token from account
+   *  - get secret and QR code image src from totp token
+   *  - add in success and error messages (with Banner component)
+   *  - add in actions for `Cancel` `Ready` etc
+   *  - fetch code here if that's preferable
+   */
+
+  const ftlMsgResolver = useFtlMsgResolver();
+  const localizedSecurityCodePlaceholder = ftlMsgResolver.getMsg(
+    'inline-totp-setup-security-code-placeholder',
```

Comment:
Spot checked this, and I didn't see any other references for "inline-totp-setup-security-code-placeholder" in this PR.
And if I check out this PR via <kbd>gh pr checkout 14822</kbd>, and do <kbd>git grep "inline-totp-setup-security-code-placeholder" packages</kbd> I only see this instance and nothing in any .ftl files.

Same w/ `inline-totp-setup-qr-code-alt-text` below. Not sure if I'm missing something.

Reply:
Thank you for catching this! I added in the existing IDs where needed and added the others to the ftl doc